.vscode/launch.json

```diff
@@ -18,6 +18,21 @@
       "internalConsoleOptions": "neverOpen",
       "port": 9229,
       "smartStep": true
+    },
+    {
+      "name": "Launch index.ts",
+      "env": {
+        "TS_NODE_COMPILER_OPTIONS": "{\"resolveJsonModule\": true}"
+      },
+      "type": "node",
+      "request": "launch",
+      "runtimeArgs": [
+        "-r",
+        "ts-node/register"
+      ],
+      "args": [
+        "${workspaceRoot}/libs/hdf_converters/src/test.ts"
+      ]
```

Comment:
This should probably be removed since I don't think the `test.ts` file is the long term way we were performing tests.